<launch>
  <!-- Joystick -->
  <node respawn="true" pkg="joy" type="joy_node" name="joy">
    <param name="dev" value="/dev/input/js0" />
    <param name="deadzone" value="0.05" />
  </node>


  <!-- Joystick Teleop -->
  <node pkg="car" type="joystick_teleop" name="joystick_teleop" output="screen" respawn="true">
    <param name="topic_stop_flag" value="/stop_flag"/>
    <param name="topic_cmd_vel_nav" value="/astar/cmd_vel"/>
    <param name="topic_cmd_vel" value="/cmd_vel_final"/>
    <param name="button_nav" value="0"/>
    <param name="button_manual" value="1"/>
    <param name="axis_linear" value="1"/>
    <param name="axis_angular" value="3"/>
    <param name="scale_linear" value="1.0"/>
    <param name="scale_angular" value="0.785398163"/>
  </node>
</launch>
